apps/www/lib/context/MeContext.tsx

```diff
@@ -135,6 +189,25 @@ const MeContextProvider = ({ children }: Props) => {
     } catch (e) {}
   }, [loading, portraitOrInitials, hasActiveMembership])
 
+  // Sync loggedIn state between multiple tabs by observing localStorage changes
+  useEffect(() => {
+    const newAuthState = getAuthState()
+    // In case the new authState diverges from the current authState,
+    // broadcast to other tabs to update their state
+    if (authState !== newAuthState) {
+      setAuthState(newAuthState)
+      broadcastAuthenticationChange(newAuthState)
+    }
+  }, [getAuthState])
+
+  // Register a storage-event listener
+  useEffect(() => {
+    window.addEventListener('storage', handleAuthenticationChangeBroadcast)
+    return () => {
+      window.removeEventListener('storage', handleAuthenticationChangeBroadcast)
+    }
+  }, [handleAuthenticationChangeBroadcast])
```

Comment:
currently local storage calls are made outside try catchs in `broadcastAuthenticationChange` which is unsafe and will explode in some browsers.

Probably better of using our existing `createPersistedState` which already does cross tab synch, ideally with `ME_PORTRAIT_STORAGE_KEY`.

Reply:
In regards to having issues in some browsers: Who ever uses a browser that doesn't support [localStorage](https://caniuse.com/?search=localstorage) in 2022 or has somehow messed up his browser-config is totally deserving off the site crashing on him ^^

Using ME_PORTRAIT_STORAGE_KEY seems very wrong to me. Especially since it means that we would effectively duplicate the portrait-state of the logged in user within the context.

What about creating a hash from the user object and sharing that?
This way we could also ensure that the sync only happens if the Me object has effectively changed.

Reply:
removed as discussed